### ISSUE TITLE
Sidebar slide over map

### ISSUE THREAD

**Issue by Monduiz** [bmcbride/bootleaf#34]
Is there a way to modify the sidebar to slide over the map? I'm working on an application where I have components to the right of the map and it would be better if they didn't get pushed to the bottom when the user wants to use the markers list.

**Comment by bmcbride** [comment]
Have a look at: https://github.com/turbo87/sidebar-v2/

**Comment by Monduiz** [comment]
Bryan, thank you. I knew about that plugin but I don't like how it looks on a map. Your implementation feels more like a web application.

Also, I tested bootleaf's functionality of a list and map with a file of 30,000 items and something is failing. When I try with a list of 2000, it works. It's not with marketClusters, the plugin can handle that many locations very well so I'm wondering if it's list.js or typehead who can't handle it.

Have you encountered this with larger files?

Thank you by the way, your work is very inspiring!

Bjenk

**Comment by bmcbride** [comment]
Hi Bjenk,

30,000 is a LOT of items to throw at the browser. How big is that GeoJSON file? I'm certainly not surprised you are having issues trying to parse that file and manage that many DOM objects. My suggestion would be to look at some server-side solution for handling this. Perhaps you could use CartoDB?

**Comment by Monduiz** [comment]
Hi Bryan, 

Yes it's a lot. I've had to work around not having access to a server for different reasons, it's complicated and related to bureaucratic politics.

I did some test and was able to get som functionality by disabling the sync sidebar function but obviously this isn't the solution.

The file is 8mg. I have tried to break down the data into chunks and then creating bloodhounds datasets and join in an array but it's not helping.

Maybe you are right, with this many, I would have to push for a server-side solution. In leaflet, simply showing the locations and info in a doc is not a problem. It's just the list that can't take it.

Thanks for the suggestions.
